### DISCUSSION TITLE
File Analytics Dashboard

### DISCUSSION THREAD

**Discussion by stas4141** [NetApp/harvest#3205] [Q&A]
Hi,
I am using the File System Analytics dashboard,
The dashboard displays the top directories growth and size.
My question is, if it's possible to add to the dashboard the **subfolders** size and growth?
We have approximately 15 folders (directories) in the top level , but also thousands of subdirectories, which are not shown in the FSA Dashboard
Is it something that can be configured In the dashboard settings?

Please advise,

**Comment by rahulguptajss** [comment]
@stas4141 We are not collecting sub-directories analytics data because retrieving each subdirectory requires individual API calls. This approach can result in a high number of API calls and increased response times, adding additional load on ONTAP. This makes it impractical if there are many subdirectories. Moreover, these calls could generate an excessive number of metrics for Prometheus.

On a separate note, we have recently implemented a feature for identifying the top clients for a volume, which you might find interesting. More details can be found [here](https://github.com/NetApp/harvest/discussions/3130).
